package.json

```diff
@@ -27,13 +27,16 @@
     "lint": "tsc --noEmit && eslint 'src/**/*.{js,ts,tsx}' --quiet --fix",
     "test": "jest --silent",
     "prepublishOnly": "npm run test && npm run build",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
```

Comment:
Love, love, love this 😍 

Fixing the deps.

Reply:
@mnemeth-opti Is there a way to re-scan to see if the bot is happy now w/ my feature branch?

Reply:
I don't think we can force it to rescan here, it would have to be on a PR. I'm not a GH expert by any stretch, I see this is a draft... I wonder if it's moved from draft to live PR (is that a thing?) it will rescan. Also, worst case, Prisma does a FULL scan 2x daily and most current status post those scans can be seen in the console (ie for this repo in the console it just won't be a finding anymore, though there should be historical data).

I can also show you in the console what this all slices and dices to if you want, it's helpful to look in there sometimes (for specific things) since that 2x a day scan will find all items, not just ones in the diff of the PR

Reply:
Thanks. Trying a close-reopen.